### PR TITLE
Published recommended driver block rules from HVCI/10S blocklist

### PR DESCRIPTION
**Pull request by jgeurten**
Added the driver block list to the recommended block list section. We can now recommend to WDAC customers to add these vulnerable driver blocks to their policy to broaden the coverage of the kernel block policy.

### CODE REVIEW

**Comment by jgeurten** [comment]
Can I please get @jsuther1974 and @degranit added to the PR as **required**  reviewers?

**Comment by illfated** [comment]
Fair request. One of the MS Docs team members will assign/request the reviewers for you.

**Review by degranit — APPROVED** [review]
Looks great!

File: windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules.md [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+- Hypervisor-protected code integrity (HVCI) enabled devices 
+- Windows 10S mode devices
+
+Microsoft recommends enabling [HVCI](https://docs.microsoft.com/en-us/windows/security/threat-protection/device-guard/enable-virtualization-based-protection-of-code-integrity) or Windows 10S mode to protect your devices against security threats. If this is not possible, Microsoft recommends blocking the following list of drivers by merging this policy with your existing Windows Defender Application Control policy. Blocking kernel drivers without sufficient testing can result in devices or software to malfunction, and in rare cases, blue screen. It is recommended to first validate this policy in [audit mode](audit-windows-defender-application-control-policies.md) and review the audit block events.
+
+
+> [!Note]
+> This application list will be updated with the latest vendor information as application vulnerabilities are resolved and new issues are discovered. It is recommended that this policy be first validated in audit mode before rolling the rules into enforcement mode. 
+
+
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules.md:38]: Redundant blank line, I believe 2 blank lines should be sufficient spacing:
```suggestion
```

**Review by illfated — APPROVED** [review]
Looks good to me, although I am not an expert on this topic.

File: windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules.md [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+**Applies to:**
+
+- Windows 10
+- Windows Server 2016 and above
+
+Microsoft has strict requirements for code running in kernel. Consequently, malicious actors are turning to exploit vulnerabilities in legitimate and signed kernel drivers to run malware in kernel. One of the many strengths of the Windows platform is our strong collaboration with independent hardware vendors (IHVs) and OEMs. Mirosoft works closely with our IHVs and security community to ensure the highest level of driver security for our customers and when vulnerabilities in drivers do arise, that they are patched and rolled out to the ecosystem in an expedited manner. Microsoft then adds the vulnerable versions of the drivers to our ecosystem block policy which is applied to the following sets of devices: 
+
+- Hypervisor-protected code integrity (HVCI) enabled devices 
+- Windows 10S mode devices
```
Comment (jsuther1974) [inline windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules.md:30]: The correct term is "Windows 10 in S mode (S mode)", not Windows 10S mode devices. Referening "(S mode)" on first use allows you to simply refer to S mode through the rest of the doc.

File: windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules.md [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+> This application list will be updated with the latest vendor information as application vulnerabilities are resolved and new issues are discovered. It is recommended that this policy be first validated in audit mode before rolling the rules into enforcement mode. 
+
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<SiPolicy xmlns="urn:schemas-microsoft-com:sipolicy">
+  <VersionEx>10.0.19565.0</VersionEx>
+  <PolicyTypeID>{D2BDA982-CCF6-4344-AC5B-0B44427B6816}</PolicyTypeID>
+  <PlatformID>{2E07F7E4-194C-4D20-B7C9-6F44A6C5A234}</PlatformID>
+  <Rules>
```
Comment (jsuther1974) [inline windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules.md:45]: Are these really the only rule options we should include by default?

**Review by jsuther1974 — CHANGES_REQUESTED** [review]
I'm not a *huge* fan of embedding the XML in a help topic rather than manifesting the XML to the Windows\Schemas folder. I'll live with it, though, if we have to...

File: windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules.md (hunk 3 above)
Comment (jgeurten) [reply windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules.md:45]: I was surprised too - instructions in the readme.txt (minkernel/ci/data/MaliciousDriverPolicy/readme.txt)

File: windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules.md (hunk 2 above)
Comment (jgeurten) [reply windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules.md]: Done!

File: windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules.md [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+---
+
+# Microsoft recommended driver block rules
+
+**Applies to:**
+
+- Windows 10
+- Windows Server 2016 and above
+
+Microsoft has strict requirements for code running in kernel. Consequently, malicious actors are turning to exploit vulnerabilities in legitimate and signed kernel drivers to run malware in kernel. One of the many strengths of the Windows platform is our strong collaboration with independent hardware vendors (IHVs) and OEMs. Mirosoft works closely with our IHVs and security community to ensure the highest level of driver security for our customers and when vulnerabilities in drivers do arise, that they are patched and rolled out to the ecosystem in an expedited manner. Microsoft then adds the vulnerable versions of the drivers to our ecosystem block policy which is applied to the following sets of devices:
```
Comment (jsuther1974) [inline windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules.md:27]: You have a typo. “Mirosoft”

**Review by jsuther1974 — APPROVED** [review]

**Pull request merged**